client/guidestours/config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
It'd be good to convert this to a `.js` config, so it's less painful to write, and gives us a bit more flexibility wrt i18n.